scipy/stats/_discrete_distns.py

```diff
@@ -449,14 +451,16 @@ def kurtosis(n, a, b):
             term = (a - 2.)
             term_2 = ((a - 1.)**2. * (a**2. + a * (6 * b - 1.)
                       + 6. * (b - 1.) * b)
-                      + 3. * n**2 * ((a + 5.) * b**2. + (a + 5.)
+                      + 3. * n**2. * ((a + 5.) * b**2. + (a + 5.)
                       * (a - 1.) * b + 2. * (a - 1.)**2)
                       + 3 * (a - 1.) * n
                       * ((a + 5.) * b**2. + (a + 5.) * (a - 1.) * b
                       + 2. * (a - 1.)**2.))
             denominator = ((a - 4.) * (a - 3.) * b * n
                            * (a + b - 1.) * (a + n - 1.))
-            return term * term_2 / denominator
+            # Wolfram Alpha uses Pearson kurtosis, so we substract 3 to get
+            # scipy's Fisher kurtosis
+            return term * term_2 / denominator - 3.
```

Comment:
I think this is wrong.
AFAIR, all distributions return pearson kurtosis. Only the stats function kurtosis defaults to excess kurtosis

Reply:
It should be ok: `stats.norm().stats('k')` returns 0 (Fisher kurtosis)